### ISSUE TITLE
How does reward correlated with the acceptance rate?

### ISSUE THREAD

**Issue by ubriela** [ubriela/geocrowd-pricing#1]
Acceptance rate is the probability the user perform the assigned task.
Some thoughts:

Option 1:
Intuitively, higher price means higher f. So f should monotonicity increase with higher price. Function like f = 1-1/(price+1) that asymptotically approaches 1.

Option 2:
We can use normal distribution, and that is called Cumulative Gaussian (Normal) Distribution.
http://stats.stackexchange.com/questions/158556/probability-of-agreeing-to-do-some-work-depending-on-the-payment, 
Basically, we can chose sigma =(p2-p1)/2 or (p2-p1)/4, or 6, etc... It is more common to use normal distribution when we talk about stimulation.
Here is the cumulative distribution function (CDF) of a Gaussian with mean μ=(p1+p2)/2 and variance σ=(p2−p1)/6, which is of course monotonic in price (as all CDFs must be):
![unnamed](https://cloud.githubusercontent.com/assets/3415436/9803154/b531eb92-57d4-11e5-94a2-2be8ca3b3dfb.png)
